components/Footer/Footer.jsx

```diff
@@ -1,5 +1,87 @@
+                        <div className={styles.logoAndSocial}>
+                            <Image src={britoFlag} alt='logo' width={72.33} height={64.36} />
+                            <Image src={britoWordmark} alt='wordmark' width={193} height={72} />
+                            <p className={'vertical-padding'}>
+                                Building the Brazilian IT community in Ottawa
+                            </p>
+                        </div>
+                        <div className={styles.socialContainer}>
+                            <a
+                                href='https://www.facebook.com/groups/566404256865424'
+                                target={'_blank'}
+                                rel='noopener noreferrer'
+                            >
+                                <Image src={fbLogo} alt='facebook' width={9.82} height={18} />
+                            </a>
+                            <a href='' target={'_blank'} rel='noopener noreferrer'>
+                                <Image src={instaLogo} alt='instagram' width={18} height={18} />
+                            </a>
+                            <a
+                                href='https://www.linkedin.com/groups/12058916/'
+                                target={'_blank'}
+                                rel='noopener noreferrer'
+                            >
+                                <Image src={linkedinLogo} alt='linkedin' width={19} height={18} />
+                            </a>
+                        </div>
+                    </div>
+                    <div></div>
+                    <nav>
+                        <h5 className={styles.title}>Company</h5>
+                        <div>
+                            <p>
+                                <a className={`${styles.navigation} body2`}>About</a>
+                            </p>
+                            <p className={'vertical-padding'}>
+                                <a className={`${styles.navigation} body2`}>Contact</a>
+                            </p>
+                        </div>
+                    </nav>
+                    <nav>
+                        <h5 className={styles.title}>Join us</h5>
+                        <p>
+                            <a className={`${styles.navigation} body2`}>Network list</a>
+                        </p>
+                    </nav>
+                </div>
+                <Image src={line} alt='divisor' width={1240} height={3} />
+                <div className={`${styles.footerBottom} vertical-padding`}>
+                    <p className={styles.copyrightText}>Copyright © 2023 Brito</p>
+                    <p></p>
+                    <div className={styles.footerBottom}>
+                        <p className={styles.copyrightText}>All Rights Reserved</p>
+                        <p>&nbsp;|&nbsp;</p>
+                        <p>
+                            <a href='' className={styles.termsLink}>
```

Comment:
Empty link

Reply:
We have a file/page with terms and privacy policy?